### PR TITLE
Draw the Frenet trihedron

### PR DESCRIPTION
**Pull request by eggrobin**
There was a bug in the plotting of the Frenet trihedron in a rotating frame that suggests we may want to distinguish `World` and `PlottedWorld` somehow, since the plotting operation, namely from `Barycentric` to `Navigation` (`plotting_frame`) at actual time and then from `Navigation` (`plotting_frame`) to `Barycentric` at current time, does not change the type.

### CODE REVIEW

File: ksp_plugin/interface_flight_plan.cpp [hunk 1]
```diff
@@ -89,19 +89,26 @@ NavigationManoeuvre ToNavigationManoeuvre(Plugin const* const plugin,
...
-      ToXYZ(world_frenet_frame(Vector<double, Frenet<Navigation>>({0, 0, 1}))
-                .coordinates());
+  Instant const current_time = plugin->CurrentTime();
+  Instant const initial_time = manœuvre.initial_time();
+  auto const plotting_frame = plugin->GetPlottingFrame();
+  OrthogonalMap<Frenet<Navigation>, World> frenet_to_plotted_world =
+      barycentric_to_world *
+      plotting_frame->FromThisFrameAtTime(current_time).orthogonal_map() *
+      plotting_frame->ToThisFrameAtTime(initial_time).orthogonal_map() *
+      frenet_to_barycentric;
```
Comment (pleroy) [inline ksp_plugin/interface_flight_plan.cpp]: Should this be merged with PlotBarycentricPosition?

File: ksp_plugin_adapter/ksp_plugin_adapter.cs [hunk 2]
```diff
@@ -781,7 +783,41 @@ where is_in_physics_bubble(vessel)
...
+            NavigationManoeuvre manoeuvre = plugin_.FlightPlanGetManoeuvre(
+                                                active_vessel_guid,
+                                                manoeuvre_index);
+            double scale = (ScaledSpace.ScaledToLocalSpace(
+                                MapView.MapCamera.transform.position) -
+                            position_at_ignition).magnitude * 0.015;
+            Func<Vector3d, UnityEngine.Vector2> world_to_screen =
+                (world_position) =>
+                    MapView.MapCamera.camera.WorldToScreenPoint(
+                          ScaledSpace.LocalToScaledSpace(world_position));
```
Comment (pleroy) [inline ksp_plugin_adapter/ksp_plugin_adapter.cs]: Bad indent.

File: ksp_plugin_adapter/ksp_plugin_adapter.cs [hunk 3]
```diff
@@ -844,11 +880,20 @@ where is_in_physics_bubble(vessel)
       rendered_flight_plan_[i] = NewRenderedTrajectory(
           (i % 2 == 0) ? XKCDColors.RoyalBlue : XKCDColors.OrangeRed);
     }
+    rendered_frenet_trihedra_ = new VectorLine[3 * (segments / 2)];
+    for (int i = 0; i < segments / 2; ++i) {
+      rendered_frenet_trihedra_[3 * i] =
+          NewUILine(new UnityEngine.Color(0.84f, 1, 0));
```
Comment (pleroy) [inline ksp_plugin_adapter/ksp_plugin_adapter.cs]: Why not xkcd colors?

Comment (eggrobin) [inline ksp_plugin_adapter/ksp_plugin_adapter.cs]: Done, it was hard to find an utility to convert RGB values to the nearest xkcd colour though.

File: ksp_plugin_adapter/ksp_plugin_adapter.cs (hunk 2 above)
Comment (eggrobin) [inline ksp_plugin_adapter/ksp_plugin_adapter.cs]: Done.

File: ksp_plugin/interface_flight_plan.cpp (hunk 1 above)
Comment (eggrobin) [inline ksp_plugin/interface_flight_plan.cpp]: This should be typed eventually anyway, making a TODO

**Pull request merged**